client/gutenberg/extensions/presets/jetpack/index.json

```diff
@@ -0,0 +1 @@
+[ "markdown", "related-posts","tiled-gallery" ]
```

Comment:
Remember to rebase, `publicize` extension [is now enabled](https://github.com/Automattic/wp-calypso/pull/28110) in Jetpack preset as well.